loadgen/README.md

```diff
@@ -1,14 +1,11 @@
 # Overview {#mainpage}
 
-*Note:* A compiled html version of this document is hosted online
-[here](https://mlperf.github.io/inference/loadgen/index.html).
-
 ## Introduction
 
 * The LoadGen is a *reusable* module that *efficiently* and *fairly* measures
   the performance of inference systems.
 * It generates traffic for scenarios as formulated by a diverse set of experts
-  in the [MLPerf working group](https://mlperf.org/about).
+  in the [MLCommons working group](https://mlperf.org/about).
```

Comment:
The link itself should also change to MLCommons:
in the [MLCommons working group](https://mlcommons.org/).

Reply:
Also change line 43:
[MLCommons Website](https://mlcommons.org/)